android/expoview/build.gradle

```diff
@@ -134,82 +132,25 @@ dependencies {
-  compileOnly 'host.exp.exponent:expo-analytics-amplitude:4.0.0'
-  compileOnly 'host.exp.exponent:expo-analytics-segment:4.0.0'
-  compileOnly 'host.exp.exponent:expo-app-auth:4.0.0'
-  compileOnly 'host.exp.exponent:expo-av:4.1.0'
-  compileOnly 'host.exp.exponent:expo-background-fetch:4.0.0'
-  compileOnly 'host.exp.exponent:expo-barcode-scanner:4.0.0'
-  compileOnly 'org.unimodules:unimodules-barcode-scanner-interface:1.0.0'
-  compileOnly 'host.exp.exponent:expo-brightness:4.0.0'
-  compileOnly 'host.exp.exponent:expo-calendar:4.0.0'
-  compileOnly 'host.exp.exponent:expo-camera:4.0.0'
-  compileOnly 'org.unimodules:unimodules-camera-interface:1.0.0'
-  compileOnly 'host.exp.exponent:expo-contacts:4.0.0'
-  compileOnly 'host.exp.exponent:expo-crypto:4.0.0'
-  compileOnly 'host.exp.exponent:expo-face-detector:4.0.0'
-  compileOnly 'host.exp.exponent:expo-facebook:4.0.0'
-  compileOnly 'host.exp.exponent:expo-font:4.0.0'
-  compileOnly 'host.exp.exponent:expo-gl:4.0.0'
-  compileOnly 'host.exp.exponent:expo-gl-cpp:4.0.0'
-  compileOnly 'host.exp.exponent:expo-google-sign-in:4.0.0'
-  compileOnly 'host.exp.exponent:expo-haptics:4.0.0'
-  compileOnly 'org.unimodules:unimodules-image-loader-interface:1.0.0'
-  compileOnly 'host.exp.exponent:expo-image-manipulator:4.0.0'
-  compileOnly 'host.exp.exponent:expo-image-picker:4.0.0'
-  compileOnly 'host.exp.exponent:expo-intent-launcher:4.0.0'
-  compileOnly 'host.exp.exponent:expo-keep-awake:4.0.0'
-  compileOnly 'host.exp.exponent:expo-linear-gradient:4.0.0'
-  compileOnly 'host.exp.exponent:expo-local-authentication:4.0.0'
-  compileOnly 'host.exp.exponent:expo-localization:4.0.0'
-  compileOnly 'host.exp.exponent:expo-location:4.0.0'
-  compileOnly 'host.exp.exponent:expo-mail-composer:4.0.0'
-  compileOnly 'host.exp.exponent:expo-media-library:4.0.0'
-  compileOnly 'host.exp.exponent:expo-payments-stripe:4.0.0'
-  compileOnly 'host.exp.exponent:expo-print:4.0.0'
-  compileOnly 'host.exp.exponent:expo-random:4.0.0'
-  compileOnly 'host.exp.exponent:expo-sensors:4.0.0'
-  compileOnly 'host.exp.exponent:expo-sharing:2.0.0'
-  compileOnly 'host.exp.exponent:expo-sms:4.0.0'
-  compileOnly 'host.exp.exponent:expo-speech:4.0.0'
-  compileOnly 'host.exp.exponent:expo-sqlite:4.0.0'
-  compileOnly 'host.exp.exponent:expo-web-browser:4.0.0'
+    addMavenUnimodulesDependencies([
+      modulesPaths : ['../../packages'],
+      configuration: 'compileOnly',
+      target       : 'react-native'
+  ])
   END UNCOMMENT WHEN DISTRIBUTING */
 
   // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
   api project(':ReactAndroid')
 
   // Universal modules
-  // They're defined as "api" so we don't have to define them again in app/build.gradle
   // In distribution they're "compileOnly" and it's the app/build.gradle who manages them.
   addUnimodulesDependencies([
       modulesPaths : ['../../packages'],
       configuration: 'api',
-      target       : 'react-native',
-      exclude      : [
-          'expo-module-template',
```

Comment:
Could you add either a corresponding PR to `xdl` that will rename the template to `unimodule.json` or an issue on the `expo-cli` repo that we should add this functionality?

Reply:
https://github.com/expo/expo-cli/pull/553

Reply:
Thanks, merged! Will you publish a new version of XDL with this patch and upgrade XDL used in `tools` and `tools-public`, please?